src/components/cylc/gscan/GScan.vue

```diff
@@ -158,70 +125,47 @@ export default {
             undefined,
             { numeric: true, sensitivity: 'base' })
       })
-    },
-    /**
-     * Compute summary information, where the key is the name of a workflow, the value is another map with the summary.
-     * @returns {Map<String, Map>}
-     */
-    workflowsSummaries () {
-      const workflowSummaries = new Map()
-      // with async scan, the workflows list may be null or undefined
-      // see cylc-uiserver PR#150
-      if (this.workflows) {
-        for (const workflow of this.workflows) {
-          workflowSummaries.set(workflow.name, getWorkflowSummary(workflow))
-        }
-      }
-      return workflowSummaries
     }
   },
   created () {
-    this.viewID = `GScan(*): ${Math.random()}`
-    this.$workflowService.register(
-      this,
-      {
-        activeCallback: this.setActive
+    const vm = this
+    this.deltasObservable = this.$workflowService.startDeltasSubscription(GSCAN_DELTAS_SUBSCRIPTION, {}, {
+      next: function next (response) {
+        const added = response.data.deltas.added
+        const updated = response.data.deltas.updated
+        const id = response.data.deltas.id
+        if (added && added.workflow && added.workflow.status) {
+          const workflow = Object.assign({}, added.workflow, vm.workflows.get(id))
+          vm.workflows.set(workflow.id, workflow)
+          vm.workflowsTracker++
```

Comment:
What is the purpose of `workflowsTracker`, I can't quite work it out.

Reply:
Vue 2 does not track reactivity of Set, Map... so you can either use a separate variable to trigger Vue's reactivity, or workaround with Vue.set, Vue.delete, maybe a watcher... in this case I thought this was the simplest solution.

And Vue3 fixes it, as you can have reactive Maps.

Reply:
Can you add a comment to the code, so we know what's going on here in future?

Reply:
There's one already where the variable is declared: https://github.com/cylc/cylc-ui/pull/543/files#diff-fbd6564546be7943d0780e9d928d7a79600e72699daed0fe29bbec1efae43a5fR95-R97

But I can add one here too, or try to improve that comment if necessary.

Reply:
Ah, OK, no worries.